hugegraph-core/src/main/java/com/baidu/hugegraph/HugeGraph.java

```diff
@@ -161,12 +163,17 @@ public boolean closed() {
         return this.closed && this.tx.closed();
     }
 
-    public boolean restoring() {
-        return this.restoring;
+    public GraphMode mode() {
+        return this.mode;
+    }
+
+    public void mode(GraphMode mode) {
+        this.mode = mode;
     }
 
-    public void restoring(boolean restoring) {
-        this.restoring = restoring;
+    public boolean restoring() {
```

Comment:
it's not used any more?

Reply:
still used in constructVertex() and checkId() of GraphTransaction